lib/Utils/EdgeFunctionStats.cpp

```diff
@@ -0,0 +1,51 @@
+#include "phasar/DataFlow/IfdsIde/EdgeFunctionStats.h"
+
+#include "llvm/ADT/STLExtras.h"
+#include "llvm/Support/Format.h"
+
+llvm::raw_ostream &psr::operator<<(llvm::raw_ostream &OS,
+                                   const EdgeFunctionStats &S) {
+  static constexpr std::array<llvm::StringRef, 5> EFKind{
+      "Normal", "Call", "Return", "CallToReturn", "Summary"};
+  static constexpr std::array<llvm::StringRef, 3> AllocKind{
+      "SmallObjectOptimized", "DefaultHeapAllocated", "CustomHeapAllocated"};
```

Comment:
would it hurt to just make it an initializer_list?

`static constexpr auto EFKind{"Normal", "Call", "Return", "CallToReturn", "Summary"};`

Reply:
Fixed